templates/docs/examples/patterns/slider/slider-input.html

```diff
@@ -15,10 +15,10 @@
     id="slider3"
     aria-label="Example slider, range 0 to 100">
   <input class="p-slider__input"
-    type="text"
+    type="number"
```

Comment:
We keep it as text to avoid having native number controls that mess up with text align:

<img width="281" alt="image" src="https://user-images.githubusercontent.com/83575/195011177-2ed633d7-43db-4dad-af28-9bc6f1f69bae.png">

Reply:
I've reverted this to text.